sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -103,8 +101,7 @@ public async Task ExtractKeyPhrasesTest()
             TextAnalyticsClient client = GetClient();
             string input = "My cat might need to see a veterinarian.";
 
-            ExtractKeyPhrasesResult result = await client.ExtractKeyPhrasesAsync(input);
-            IReadOnlyCollection<string> keyPhrases = result.KeyPhrases;
+            IList<string> keyPhrases = (IList<string>)await client.ExtractKeyPhrasesAsync(input);
```

Comment:
@pakrym what is the correct way to call this here?  `Response<IList<string>> keyPhrases = await client.ExtractKeyPhrasesAsync(input);` ?

Reply:
I prefer the `Response<IList<string>> keyPhrases = await client.ExtractKeyPhrasesAsync(input);` options.